### ISSUE TITLE
License details, commercial use of the model

### ISSUE THREAD

**Issue by margokhokhlova** [Deci-AI/super-gradients#1174]
### 💡 Your Question

Hello!
Please, can you confirm that the model architecture and the code for training are free for the commercial use?
From the description in the license Section, I understand that only the weights are under a more restrictive license.
Please, can you confirm that the model trained on custom data cat be used in a commercial product? 
Thank you very much.

**Comment by ofrimasad** [comment]
Hi @margokhokhlova.
I confirm that the code and architecture are free for commercial use.
If you train your model on custom data, starting from random weights, it will not be restricted for commercial use.
Only if you use our pre-trained weights to start your training, you will have commercial limitations

**Comment by harpreetsahota204** [comment]
Hi @margokhokhlova 

Thanks for opening an issue for SG. I'm gathering some feedback on SuperGradients and YOLO-NAS.

Would you be down for a quick call to chat about your experience?

If a call doesn't work for you, no worries. I've got a short survey you could fill out: https://bit.ly/sgyn-feedback.

I know you’re super busy, but your input will help us shape the direction of SuperGradients and make it as useful as possible for you.

I appreciate your time and feedback. Let me know what works for you.

Cheers,

Harpreet

**Comment by quangsonle** [comment]
> Hi @margokhokhlova. I confirm that the code and architecture are free for commercial use. If you train your model on custom data, starting from random weights, it will not be restricted for commercial use. Only if you use our pre-trained weights to start your training, you will have commercial limitations

@ofrimasad  please confirm if I declare a model to train with this:

`model = models.get(Models.YOLOX_S, pretrained_weights="coco")`

, it means I use the pre-trained weights as you mentioned. So if I do like this:

`model = models.get(Models.YOLOX_S)`

, I will be free of charge.

Otherwise, could you please advise on how to declare the model from scratch to not be bound by the pre-trained license?
